lib/components/Dropdown/index.tsx

```diff
@@ -35,7 +36,7 @@ export const Dropdown = ({
     list,
     listHeight = 'md',
     name,
-    onChange,
+    onInputChange,
```

Comment:
If `onInputChange` is a rename of a prop, `onChange` then you can do it like this:
```suggestion
    onChange: onInputChange,
```